docs/services/general-info/env-var-deltas/4.0.0-5.0.0-added.md

```diff
@@ -27,6 +23,10 @@
+| services/auth-service/pkg/config/tracing.go   | `OCIS_TRACING_ENABLED;AUTH_SERVICE_TRACING_ENABLED`                                                                                                                                                | Activates tracing.                                                                                                                                                                                                                                                                                                                                                                                                                                                          |                                       |
+|                                               | `OCIS_TRACING_TYPE;AUTH_SERVICE_TRACING_TYPE`                                                                                                                                                      | The type of tracing. Defaults to '', which is the same as 'jaeger'. Allowed tracing types are 'jaeger' and '' as of now."`                                                                                                                                                                                                                                                                                                                                                  |                                       |
+|                                               | `OCIS_TRACING_ENDPOINT;AUTH_SERVICE_TRACING_ENDPOINT`                                                                                                                                              | The endpoint of the tracing agent.                                                                                                                                                                                                                                                                                                                                                                                                                                          |                                       |
+|                                               | `OCIS_TRACING_COLLECTOR;AUTH_SERVICE_TRACING_COLLECTOR`                                                                                                                                            | The HTTP endpoint for sending spans directly to a collector, i.e. http://jaeger-collector:14268/api/traces. Only used if the tracing endpoint is unset.                                                                                                                                                                                                                                                                                                                     |                                       |
```

Comment:
Note: these env vars are actually in `tracing.go` so I have adjusted the md file.
FYI @mmattel @dragonchaser 
If I find other minor things in the md file, I will push those also.